setup.py

```diff
@@ -87,6 +87,7 @@ def get_version():
     include_package_data=True,
     install_requires=[
         "numpy >= 1.21.0",
+        "brax-jumpy >= 0.1.0",
```

Comment:
We have moved the module to `jax-jumpy`

Reply:
https://github.com/Farama-Foundation/Gymnasium/pull/150/commits/4a33977ee593feac1818d82e02721cd599a00eb9